code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -694,3 +694,28 @@
 	id = "blank_paper"
 	results = list("blank_paper" = 3)
 	required_reagents = list("silencer" = 1, "nothing" = 1, "nuka_cola" = 1)
+
+
+/datum/chemical_reaction/wizz_fizz
+	name = "Wizz Fizz"
+	id = "wizz_fizz"
+	results = list("wizz_fizz" = 3)
+	required_reagents = list("triple_sec" = 1, "sodawater" = 1, "champagne" = 1)
+	mix_message = "The beverage starts to froth with an almost mystical zeal!"
+	mix_sound = 'sound/effects/bubbles2.ogg'
+
+
+/datum/chemical_reaction/bug_spray
+	name = "Bug Spray"
+	id = "bug_spray"
+	results = list("bug_spray" = 5)
+	required_reagents = list("triple_sec" = 2, "lemon_lime" = 1, "rum" = 2, "vodka" = 1)
```

Comment:
Is it intentional that 6u of required reagents are consumed per (only) 5u of bug spray that's created?

Reply:
Yes; multiples of 5 are nicer for filling glasses and shakers than six, but having the ratio be 2;1 for alcohols to juice is easier for dispensers